code/game/dna/mutations/mutation_powers.dm

```diff
@@ -508,11 +508,20 @@
 		user.layer = 9
 
 		user.flying = TRUE
-		for(var/i=0, i<10, i++)
+		for(var/i in 1 to 10)
```

Comment:
Could you extract this `10` value out to the spell datum itself? It's kind of a magic number, and this would also make it a little easier to adminbus.